src/examples/forms/multithumbrange/MultiThumbRangeValidation.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState, useCallback } from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { Field, Label, Message, MultiThumbRange } from '@zendeskgarden/react-forms';
+
+type validationTypes = 'success' | 'warning' | 'error';
+
+const messages: Record<validationTypes, string> = {
+  success: 'Growing all the time',
+  warning: 'Growing regularly',
+  error: 'Growing slowly'
+};
```

Comment:
With this only being referenced once, you may be able to include a memoized value in the example. Something like:

```tsx
const validationMessage = useMemo(() => {
    if (validation === 'success') {
        return 'Success message';
    } else if (validation === 'warning') {
        return 'Warning message';
    }

    return 'Error message';
}, [validation]);

return (
    <Field>
        ....
        <Message validation={validation}>{validationMessage}</Message>
    </Field>
);
```

Reply:
Wow, nice. Ty 🏆